barretenberg/cpp/src/barretenberg/stdlib/translator_vm_verifier/translator_recursive_verifier.cpp

```diff
@@ -181,6 +182,17 @@ void TranslatorRecursiveVerifier_<Flavor>::verify_translation(
     // multiply by x here to deal with shift
     eccvm_opening.assert_equal(x * accumulated_result);
 }
+
+template <typename Flavor>
+void TranslatorRecursiveVerifier_<Flavor>::verify_consistency_with_final_merge(
+    const std::array<Commitment, TranslatorFlavor::NUM_OP_QUEUE_WIRES> merge_commitments)
+{
+    // Check the consistency with final merge
+    for (auto [merge_commitment, translator_commitment] : zip_view(merge_commitments, op_queue_commitments)) {
+        merge_commitment.x.assert_equal(translator_commitment.x);
```

Comment:
Can you not assert equality on the commitments directly?

Reply:
nope, I guess we never needed it, will add one in a follow-up PR